### PR TITLE
xrootd voms authorization improvements

### PR DESCRIPTION
**Pull request by matyasselmeci**
https://opensciencegrid.atlassian.net/browse/SOFTWARE-5355

### CODE REVIEW

File: docs/data/xrootd/xrootd-authorization.md [hunk 1]
```diff
@@ -184,14 +185,24 @@ For example,
 ```
 will map FQANs starting with `/osg/` to the user `osg01`.
 
+To enable using VOMS mapfiles in the first place, add the following line to your XRootD configuration:
+```
+voms.mapfile /etc/grid-security/voms-mapfile
+```
+replacing `/etc/grid-security/voms-mapfile` with the actual location of your mapfile, if it is different.
```
Comment (brianhlin) [inline docs/data/xrootd/xrootd-authorization.md:192]:
```suggestion
To enable using VOMS mapfiles in the first place, add the following line to your XRootD configuration:
```
voms.mapfile /etc/grid-security/voms-mapfile
```
replacing `/etc/grid-security/voms-mapfile` with the actual location of your mapfile, if it is different.
```

I had a comment saying that we do this in `osg-xrootd` but we don't 🤦🏽

**Review by brianhlin — APPROVED** [review]
LGTM

File: docs/data/xrootd/xrootd-authorization.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/data/xrootd/xrootd-authorization.md:192]: yep, we don't.

Comment (brianhlin) [reply docs/data/xrootd/xrootd-authorization.md:192]: we should =/

**Pull request merged**